pkg/resource/obtenantbackuppolicy_task.go

```diff
@@ -471,7 +479,7 @@ func (m *ObTenantBackupPolicyManager) getOperationManager() (*operation.Oceanbas
 		tenantCR := &v1alpha1.OBTenant{}
 		err = m.Client.Get(m.Ctx, types.NamespacedName{
 			Namespace: m.BackupPolicy.Namespace,
-			Name:      m.BackupPolicy.Spec.TenantName,
```

Comment:
My mistake. If the tenant CR has the same metadata.name and spec.tenantName, this bug can not be found in dev testings.